test/models/item_test.rb

```diff
@@ -1,7 +1,20 @@
 require 'test_helper'
 
 class ItemTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+
+  def valid_item
+    Item.new(title: "Hot Coffee", description: "ground heaven", price: 2, image: "http://i.dailymail.co.uk/i/pix/2013/10/24/article-0-16B39191000005DC-568_634x403.jpg")
+  end
+
+  test 'is valid' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.